content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -447,5 +501,4 @@ be found under :menuselection:`Accounting --> Reporting --> Colombian Statements
 Retención en Fuente`.
 
 .. image:: colombia/fuente-report.png
```

Comment:
@samueljlieber Change to `retencion-fuente-dian.png` already in [Google Drive folder](https://drive.google.com/drive/folders/11cXF1gXxwws_P9hUslnSJcfjheEbBJZN) :)